### ISSUE TITLE
Triple quote doesn't mix with single quote

### ISSUE THREAD

**Issue by briansfrank** [Novo-Int/xeto-vscode-extension#48]
This Xeto snippet is not handling nested single quotes:

```
accessibility: Icon {
  svg:"""
      <svg
        xmlns="http://www.w3.org/2000/svg"
        width="24"
        height="24"
        viewBox="0 0 24 24"
        fill="none"
        stroke="currentColor"
        stroke-width="2"
        stroke-linecap="round"
        stroke-linejoin="round"
      >
        <circle cx="16" cy="4" r="1" />
        <path d="m18 19 1-7-6 1" />
        <path d="m5 8 3-3 5.5 3-2.36 3.5" />
        <path d="M4.24 14.5a5 5 0 0 0 6.88 6" />
        <path d="M13.76 17.5a5 5 0 0 0-6.88-6" />
      </svg>
      """
}
```

Edit: Actually maybe you don't support triple multi-line strings at all is the problem...

**Comment by hades2510** [comment]
this should work in 0.1.28.

The compiler supported triple quotes, but the textmate grammar wasn't updated.
At the moment the content of a triple quotes string has a different color than a string.

We may want to add support for embedded languages, but at the moment we don't support this.
If we decide to add support for embedded languages we need a way of knowing the language. We may relay on some heuristics to figure out if it's XML, HTML, SVG for instance, but that's not very reliable.

A better approach would be to hint to the desired language, either as a top level comment in the comment (but comments aren't consistent across languages), or use something like this:

```
foo: {
 embedded: CodeBlock <svg> """
  <svg
        xmlns="http://www.w3.org/2000/svg"
        width="24"
        height="24"
      >
        <circle cx="16" cy="4" r="1" />
        <path d="m18 19 1-7-6 1" />
        <path d="m5 8 3-3 5.5 3-2.36 3.5" />
        <path d="M4.24 14.5a5 5 0 0 0 6.88 6" />
        <path d="M13.76 17.5a5 5 0 0 0-6.88-6" />
      </svg>
 """
  trigger: CodeBlock <javascript> """
     function cb() {
     }
  """
}
```

**Comment by rracariu** [comment]
In 0.1.28 looks that some random html tags would mess up the syntax coloring:
<img width="309" alt="image" src="https://github.com/Novo-Int/xeto-vscode-extension/assets/14125806/64872a4b-a055-4b91-83ea-742bf26f79f3">

You can see the color of closing quote and the following expressions becoming white.

**Comment by hades2510** [comment]
this should be fixed in 0.1.29

**Comment by rracariu** [comment]
tested with 0.1.29
